unraid_mcp/subscriptions/manager.py

```
@@ -153,6 +154,16 @@ async def _subscription_loop(self, subscription_name: str, query: str, variables
                 logger.debug(f"[WEBSOCKET:{subscription_name}] Connecting to: {ws_url}")
                 logger.debug(f"[WEBSOCKET:{subscription_name}] API Key present: {'Yes' if UNRAID_API_KEY else 'No'}")
 
+                # Build SSL context for wss:// connections
+                ssl_context = None
+                if ws_url.startswith('wss://'):
+                    if isinstance(UNRAID_VERIFY_SSL, str):
+                        ssl_context = ssl.create_default_context(cafile=UNRAID_VERIFY_SSL)
+                    elif UNRAID_VERIFY_SSL:
+                        ssl_context = ssl.create_default_context()
+                    else:
+                        ssl_context = ssl._create_unverified_context()
```

Ruff (0.15.0)

[error] Python allows using an insecure context via the `_create_unverified_context` that reverts to the previous behavior that does not validate certificates or perform hostname checks.

(S323)